content/history/rosters.inc

```diff
@@ -5,7 +5,8 @@
 
```

Comment:
One thing I noticed about how this file is used in this PR is that the navbar changes in a surprising way between /raceday and landing on this page -- it is, of course, worse while this page is effectively a 404.  The best fix isn't super obvious to me and is probably fine as it is.

Reply:
Yeah I don't love that either 😞 It could be interesting to figure out how to dynamically change the header based on where you're coming from? Ohh could we have a file `content/raceday/rosters.inc` that just includes this file and write the appropriate rules into `.htaccess` and `index.php`? How horrible is that 😂

Reply:
Lol I tried this and it works.. but only to display whatever the "default" year is.

https://github.com/CMUBuggy/cmubuggy.org/commit/cdc6ee91975d3046cd42f23e137a3fe06bcbedda

Reply:
I didn't see content/raceday/rosters.inc in the change?